pkg/tool/helm.go

```diff
@@ -102,8 +102,9 @@ func (h Helm) Test(namespace, release string) error {
 	utils.LogInfo(fmt.Sprintf("Execute helm test. namespace: %s, release: %s, args: %+v", namespace, release, h.args))
 	client := action.NewReleaseTesting(h.config)
 	client.Namespace = namespace
+	client.Timeout = 30 * time.Minute
```

Comment:
Right now seems to be timing out at 5 minutes, is the intention to give it more time to test? This will delay our checks by quite a bit on some cases, do you think this will impact our test harness on certification flow?

Reply:
Yes the timeout is set to prevent a false failure if the test took between 5-30 minutes. Difficult to know how long a test will take but for 99% of PR's I suspect the timeout will be irrelevant because they will complete or fail fast, just gives a test plenty of time to complete before we think it is a hang. Can reduce if you prefer.

Reply:
I think is a good start, thanks for explaining